src/lib/components/forms/form/form-components/suggestion/suggestion.component.spec.ts

```diff
@@ -1,28 +1,146 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { SuggestionComponent } from './suggestion.component';
-import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { MatAutocompleteHarness } from '@angular/material/autocomplete/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { FormsModule } from '@angular/forms';
 import { MatInputModule } from '@angular/material/input';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { TerraPlacementEnum } from '../../../../../helpers/enums/terra-placement.enum';
+import { By } from '@angular/platform-browser';
+import { MockTooltipDirective } from '../../../../../testing/mock-tooltip.directive';
+import { MatFormFieldHarness } from '@angular/material/form-field/testing';
+import { TerraSuggestionBoxValueInterface } from '../../../suggestion-box/data/terra-suggestion-box.interface';
+import { MatInputHarness } from '@angular/material/input/testing';
 
+// tslint:disable-next-line:max-function-line-count
 describe('SuggestionComponent', () => {
     let component: SuggestionComponent;
     let fixture: ComponentFixture<SuggestionComponent>;
+    let loader: HarnessLoader;
+    let autoComplete: MatAutocompleteHarness;
+    let input: MatInputHarness;
+
+    const suggestionOption1: TerraSuggestionBoxValueInterface = {
+        caption: 'Apple',
+        value: 1
+    };
+
+    const suggestionOption2: TerraSuggestionBoxValueInterface = {
+        caption: 'Banana',
+        value: 2
+    };
+
+    const suggestionOption3: TerraSuggestionBoxValueInterface = {
+        caption: 'Raspberries',
+        value: 3
+    };
+
+    const suggestionOptions: Array<TerraSuggestionBoxValueInterface> = [
+        suggestionOption1,
+        suggestionOption2,
+        suggestionOption3
+    ];
 
     beforeEach(async () => {
-        await TestBed.configureTestingModule({
-            imports: [MatAutocompleteModule, MatInputModule, MatFormFieldModule, NoopAnimationsModule],
-            declarations: [SuggestionComponent]
-        }).compileComponents();
-    });
+        TestBed.configureTestingModule({
+            imports: [MatFormFieldModule, MatInputModule, NoopAnimationsModule, FormsModule, MatAutocompleteModule],
+            declarations: [SuggestionComponent, MockTooltipDirective]
+        });
 
-    beforeEach(() => {
         fixture = TestBed.createComponent(SuggestionComponent);
         component = fixture.componentInstance;
+        loader = TestbedHarnessEnvironment.loader(fixture);
+        autoComplete = await loader.getHarness(MatAutocompleteHarness);
+        input = await loader.getHarness(MatInputHarness);
```

Comment:
Ich denke du brauchst hier nicht beides. Wenn ich das nämlich in den docs richtig verstanden habe, kannst du über die MatAutocompleteHarness mit beiden teilen (also dem input als auch dem panel) interagieren.
```suggestion
        autoComplete = await loader.getHarness(MatAutocompleteHarness);
```

Reply:
brauche den input um den isRequired zu checken, sonst weiß ich nicht wie ich drauf zugreifen kann.
        `expect(await autoComplete.isRequired()).toBe(true);`
funzt nicht

Reply:
Ok. Dann lad dir es auch nur in dem Test.
In manchen Tests checkst du auch doppelt, also auf dem autocomplete und auf dem input. Das brauchst du nicht.